components/button/button.js

```diff
@@ -41,7 +41,7 @@ class Button extends ButtonMixin(LitElement) {
 
 				button {
 					font-family: inherit;
-					padding: 0 1.5rem;
+					padding: var(--d2l-button-padding, 0 1.5rem);
```

Comment:
Wondering if we should convert this to logical if we're going to expose a CSS variable for it? Would folks ever want to tweak the inline start/end padding separately?

Reply:
Yeah good point.

Reply:
Updated. Only the inline are exposed as properties, but we can add block later if needed.